### ISSUE TITLE
Lists, when adding actors

### ISSUE THREAD

**Issue by 4C3T** [Radarr/Radarr#5693]
When adding potentially new lists of actors and such, it would be nice to have an icon (In cast-tab on the page that of a movie) that informs if that already exists, and if/when clicked (if it exists) it should retrieve the existing settings in case you want to review and/or change something.

It would also be extremely useful with a page that gathers all info about which movies the actor in question have starred, directed, produced etc.
That would be particularly useful if collections are added, so that one doesn't have to go to tmdb to look it up. Tedious process just to check up since Radarr basically doing it already.

**Comment by Qstick** [comment]
> When adding potentially new lists of actors and such, it would be nice to have an icon (In cast-tab on the page that of a movie) that informs if that already exists, and if/when clicked (if it exists) it should retrieve the existing settings in case you want to review and/or change something.


This is already there


> It would also be extremely useful with a page that gathers all info about which movies the actor in question have starred, directed, produced etc.
> That would be particularly useful if collections are added, so that one doesn't have to go to tmdb to look it up. Tedious process just to check up since Radarr basically doing it already.

You can see all this in discover

**Comment by SugarDroid** [comment]
> it would be nice to have an icon (In cast-tab on the page that of a movie) that informs if that already exists, and if/when clicked (if it exists) it should retrieve the existing settings in case you want to review and/or change something.

The icon itself is already there but invisible until your cursor don't hover over the picture of a person, and does exactly that if you click on it.
![image](https://user-images.githubusercontent.com/56313144/103713028-84c65080-4fbb-11eb-8a28-912e0026df95.png)

So i would change your request from "have an icon" to make the icon visible -simple by looking at those pictures- if the list already exist.

> It would also be extremely useful with a page that gathers all info about which movies the actor in question have starred, directed, produced etc.

Discover > Filter > Custom filter
![image](https://user-images.githubusercontent.com/56313144/103716787-25b90980-4fc4-11eb-9e71-c78f5075f9d9.png)

> That would be particularly useful if collections are added, so that one doesn't have to go to tmdb to look it up. Tedious process just to check up since Radarr basically doing it already.

I think the problem with this -if it's still about persons- is that collections on TMDb are related to movies, not persons.

**Comment by 4C3T** [comment]
Ok. My bad... Never used that "Discover" until now.
But, I think that it could do with some improvements, like a sub-menu for collections gathered from the db, right there on the page without having to be manually added, at least for  a quick glance.

Also, I don't know if you fixed any bugs recently related, but when I wrote that, it didn't update the icons I mentioned.
Neither when added or re-visiting a page.
It does now though.
I also sometimes had trouble choosing which root-folder to download to. I had to cancel then circle around a bit, then I could add folder and it would work.
At least it works at the time writing this.

Had some trouble with other pages so it could be related to my browser (Firefox). For instance, I'm unable to login to a forum run located locally (PHPbb), but on a public IP, but in Private mode it works just fine =0/
